plugin/db/mysql/mysql.go

```diff
@@ -173,6 +182,20 @@ func (driver *Driver) Execute(ctx context.Context, statement string) error {
 	return err
 }
 
+// GetLastTxnConnID returns the thread ID when executing the last transaction.
+func (driver *Driver) GetLastTxnConnID() string {
```

Comment:
Does the connection ID (or thread_id) ever change? Any reason to name it GetLastTxConnID instead of GetConnectionID() or GetThreadID()?

Reply:
It can change. The underlying connection pool could have multiple connections, thus multiple connection IDs.